src/components/BookForm.js

```diff
@@ -0,0 +1,90 @@
+/* eslint-disable class-methods-use-this */
+/* eslint-disable react/no-unescaped-entities */
+// The form should contain a
+// book title input,
+// a select box with categories and a
+// submit button.
+import React from 'react';
+import './BookForm.css';
+
+// eslint-disable-next-line no-unused-vars
+class BookForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      title: '',
+      completed: true,
+      category: '',
+    };
+    this.handleChange = this.handleChange.bind(this);
+  }
+
+  handleSubmit(event) {
+    event.preventDefault();
+    const { target } = event;
+    const title = target.title.value;
+    const category = target.category.value;
+    const completed = target.completed.value;
+    // console.log(event.target);
+    // eslint-disable-next-line no-console
+    console.clear();
+    // eslint-disable-next-line no-console
+    console.log(title, '<- title');
+    // eslint-disable-next-line no-console
+    console.log(category, '<- category');
+    // eslint-disable-next-line no-console
+    console.log(completed, '<- completed');
```

Comment:
- [ ] We need to delete all commented code and console logs here. 

- [ ] As it said in the instruction BookForm component should be placed in the `containers/` directory. The reason is that it's going to connect with Redux, therefore, it's considered a container component.
![Screenshot from 2021-01-12 20-19-02](https://user-images.githubusercontent.com/30977845/104386240-817b1980-5513-11eb-979e-7b1b38532f52.png)